core/modules/server/routes/get-status.js

```diff
@@ -17,6 +17,8 @@ exports.method = "GET";
 exports.path = /^\/status$/;
 
 exports.handler = function(request,response,state) {
+	$tw.perf.reset();
```

Comment:
Resetting the performance statistics on each request makes it impossible to analyse timings over multiple runs.

Reply:
Server timing is only about performance in a single request. For other perf measurements, we can add some other things for it later, which might only output logs on the server side.

Reply:
Maybe you mean this might clear other measurement. Do you think I should move server timing related code to a new util, so it won't affect other perf things?